hera_pspec/testing.py

```diff
@@ -401,22 +527,87 @@ def gauss_cov_fg(cov_amp, cov_length_scale, freqs, Ntimes=100, constant_in_time=
     # generate a random process from a Gaussian covariance
     Nfreqs = len(freqs)
 
-    C = cov_amp * np.exp(-((freqs[:, None] - freqs[None, :]) / cov_length_scale)**2)  # a covariance model
+    C = cov_amp * np.exp(
+        -(((freqs[:, None] - freqs[None, :]) / cov_length_scale) ** 2)
+    )  # a covariance model
 
     if constant_in_time:
         s = stats.multivariate_normal.rvs(np.zeros(Nfreqs), C, 2).reshape(2, 1, Nfreqs)
         s = s[0] + 1j * s[1]
         s = np.repeat(s, Ntimes, axis=0)
 
     else:
-        s = stats.multivariate_normal.rvs(np.zeros(Nfreqs), C, 2 * Ntimes).reshape(2, Ntimes, Nfreqs)
+        s = stats.multivariate_normal.rvs(np.zeros(Nfreqs), C, 2 * Ntimes).reshape(
+            2, Ntimes, Nfreqs
+        )
         s = s[0] + 1j * s[1]
 
     return s
 
 
-def sky_noise_sim(data, beam, cov_amp=1000, cov_length_scale=10, constant_per_bl=True,
-                  constant_in_time=True, bl_loop_seed=None, divide_by_nsamp=False):
+def sky_noise_jy_autos(lsts, freqs, autovis, omega_p, integration_time, channel_width=None, Trx=0.0):
```

Comment:
Could be worth adding to noise.py or utils.py, as it could be useful outside of testing. See `variance_from_auto_correlations` in utils.py

Reply:
I think this is OK here for now as it's legacy code supporting some tests; we can eventually replace it with an alternative in hera_sim or similar.